app/controllers/users_controller.rb

```diff
@@ -0,0 +1,47 @@
+class UsersController < ApplicationController
+  def index
+  end
```

Comment:
Use single line syntax for empty methods.
```suggestion
  def index; end
```

Reply:
By the way why do we need index action?

Reply:
Index page is the first page where the login and signup option is displayed
So to render index.html.erb I have made this method